### ISSUE TITLE
fix paths in documentation

### ISSUE THREAD

**Issue by codemzs** [dotnet/machinelearning#3474]
# Incorrect sample path

File | Message | Fix
-- | -- | --
dotnet/api/Microsoft.ML.BinaryClassificationCatalog.CalibratorsCatalog.yml | Cannot resolve '~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/BinaryClassification/Calibrators/Isotonic.cs' relative to 'api/Microsoft.ML.BinaryClassificationCatalog.CalibratorsCatalog.yml'. |  
dotnet/api/Microsoft.ML.BinaryClassificationCatalog.CalibratorsCatalog.yml | Cannot resolve '~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/BinaryClassification/Calibrators/Naive.cs' relative to 'api/Microsoft.ML.BinaryClassificationCatalog.CalibratorsCatalog.yml'. |  
dotnet/api/Microsoft.ML.BinaryClassificationCatalog.CalibratorsCatalog.yml | Cannot resolve '~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/BinaryClassification/Calibrators/FixedPlatt.cs' relative to 'api/Microsoft.ML.BinaryClassificationCatalog.CalibratorsCatalog.yml'. |  
dotnet/api/Microsoft.ML.BinaryClassificationCatalog.CalibratorsCatalog.yml | Cannot resolve '~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/BinaryClassification/Calibrators/Platt.cs' relative to 'api/Microsoft.ML.BinaryClassificationCatalog.CalibratorsCatalog.yml'. |  
dotnet/api/Microsoft.ML.Transforms.NormalizingTransformer.AffineNormalizerModelParameters-1.yml | Cannot resolve '~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Normalizer.cs' relative to 'api/Microsoft.ML.Transforms.NormalizingTransformer.AffineNormalizerModelParameters-1.yml'. |  
dotnet/api/Microsoft.ML.Transforms.NormalizingTransformer.CdfNormalizerModelParameters-1.yml | Cannot resolve '~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Normalizer.cs' relative to 'api/Microsoft.ML.Transforms.NormalizingTransformer.CdfNormalizerModelParameters-1.yml'. |  
dotnet/api/Microsoft.ML.ConversionsCatalog.yml | Cannot resolve '~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversions/MapKeyToBinaryVector.cs' relative to 'api/Microsoft.ML.ConversionsCatalog.yml'. |  
dotnet/api/Microsoft.ML.ConversionsExtensionsCatalog.yml | Cannot resolve '~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversions/MapKeyToVectorMultiColumn.cs' relative to 'api/Microsoft.ML.ConversionsExtensionsCatalog.yml'. |  
dotnet/api/Microsoft.ML.ConversionsExtensionsCatalog.yml | Cannot resolve '~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversions/MapKeyToVector.cs' relative to 'api/Microsoft.ML.ConversionsExtensionsCatalog.yml'. |  
dotnet/api/Microsoft.ML.ConversionsExtensionsCatalog.yml | Cannot resolve '~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/MappValueIdvLookup.cs' relative to 'api/Microsoft.ML.ConversionsExtensionsCatalog.yml'. |  
dotnet/api/Microsoft.ML.ConversionsExtensionsCatalog.yml | Cannot resolve '~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/MappValueToArray.cs' relative to 'api/Microsoft.ML.ConversionsExtensionsCatalog.yml'. |  
dotnet/api/Microsoft.ML.ConversionsExtensionsCatalog.yml | Cannot resolve '~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversions/MapValueToKeyMultiColumn.cs' relative to 'api/Microsoft.ML.ConversionsExtensionsCatalog.yml'. |  
dotnet/api/Microsoft.ML.StandardTrainersCatalog.yml | Cannot resolve '~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsPoissonMaximumEntropyWithOptions.cs' relative to 'api/Microsoft.ML.StandardTrainersCatalog.yml'. |  
dotnet/api/Microsoft.ML.StandardTrainersCatalog.yml | Cannot resolve '~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsPoissonMaximumEntropy.cs' relative to 'api/Microsoft.ML.StandardTrainersCatalog.yml'. |  
dotnet/api/Microsoft.ML.Trainers.KMeansModelParameters.yml | Cannot resolve '~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/KMeans.cs' relative to 'api/Microsoft.ML.Trainers.KMeansModelParameters.yml'. |  
dotnet/api/Microsoft.ML.Transforms.TensorFlowModel.yml | Cannot resolve '~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlowTransform.cs' relative to 'api/Microsoft.ML.Transforms.TensorFlowModel.yml'. |  

# Incorrect xref link

File | Message | fix
-- | -- | --
dotnet/api/Microsoft.ML.Transforms.CountFeatureSelectingEstimator.yml | 1 invalid cross reference(s) ""<xref:Microsoft.ML.Transforms.SlotsDroppingTransformer>"". |  
dotnet/api/Microsoft.ML.Transforms.CustomMappingEstimator-2.yml | 1 invalid cross reference(s) ""<xref:Microsoft.ML.Transforms.CustomMappingTransformer>"". |  
dotnet/api/Microsoft.ML.Transforms.MissingValueIndicatorEstimator.yml | 1 invalid cross reference(s) ""<xref:Microsoft.ML.Transfroms.MissingValueIndicatorTransformer/>"". |  
dotnet/api/Microsoft.ML.Transforms.Text.CustomStopWordsRemovingEstimator.yml | 1 invalid cross reference(s) ""<xref:Microsoft.ML.Transforms.Text.CustomStopWordsRemovingTransformer/>"". |  
dotnet/api/Microsoft.ML.Transforms.Text.NgramHashingEstimator.yml | 1 invalid cross reference(s) ""<xref:Microsoft.ML.Transforms.Text.NgramHashingTransformer/>"". |  
dotnet/api/Microsoft.ML.Transforms.Text.StopWordsRemovingEstimator.yml | 1 invalid cross reference(s) ""<xref:Microsoft.ML.Transforms.Text.StopWordsRemovingTransformer/>"". |  
dotnet/api/Microsoft.ML.Transforms.Text.TextFeaturizingEstimator.yml | 2 invalid cross reference(s) "Microsoft.ML.Transforms.TextFeaturizingEstimator.Options.WordFeatureExtractor", "<xref:Microsoft.ML.Transforms.TextFeaturizingEstimator.Options>". |  

# Incorrect cref link

File | Message |  
-- | -- | --
dotnet/api/Microsoft.ML.Calibrators.FixedPlattCalibratorEstimator.yml | 3 invalid cross reference(s) "Microsoft.ML.Data.DefaultColumnNames.Probability", "Microsoft.ML.Data.DefaultColumnNames.Score", "Microsoft.ML.Data.AnnotationUtils.GetTrainerOutputAnnotation(System.Boolean)". |  
dotnet/api/Microsoft.ML.Calibrators.IsotonicCalibratorEstimator.yml | 3 invalid cross reference(s) "Microsoft.ML.Data.DefaultColumnNames.Probability", "Microsoft.ML.Data.DefaultColumnNames.Score", "Microsoft.ML.Data.AnnotationUtils.GetTrainerOutputAnnotation(System.Boolean)". |  
dotnet/api/Microsoft.ML.Calibrators.NaiveCalibratorEstimator.yml | 3 invalid cross reference(s) "Microsoft.ML.Data.DefaultColumnNames.Probability", "Microsoft.ML.Data.DefaultColumnNames.Score", "Microsoft.ML.Data.AnnotationUtils.GetTrainerOutputAnnotation(System.Boolean)". |  
dotnet/api/Microsoft.ML.Calibrators.PlattCalibratorEstimator.yml | 3 invalid cross reference(s) "Microsoft.ML.Data.DefaultColumnNames.Probability", "Microsoft.ML.Data.DefaultColumnNames.Score", "Microsoft.ML.Data.AnnotationUtils.GetTrainerOutputAnnotation(System.Boolean)". |  
dotnet/api/Microsoft.ML.CustomMappingCatalog.yml | 1 invalid cross reference(s) "Microsoft.ML.Runtime.ComponentCatalog.RegisterAssembly(System.Reflection.Assembly,System.Boolean)". |  
dotnet/api/Microsoft.ML.Data.ConfusionMatrix.yml | 1 invalid cross reference(s) ""Microsoft.ML.Data.ConfusionMatrix.PredictedClassesIndicators"". |  
dotnet/api/Microsoft.ML.Data.IRowToRowMapper.yml | 1 invalid cross reference(s) ""Microsoft.ML.Data.ISchemaBoundRowMapper"". |  
dotnet/api/Microsoft.ML.Data.yml | 1 invalid cross reference(s) ""Microsoft.ML.Data.ISchemaBoundRowMapper"". |  
dotnet/api/Microsoft.ML.ModelSaveContext.yml | 1 invalid cross reference(s) ""Microsoft.ML.ICanSaveInBinaryFormat"". |  
dotnet/api/Microsoft.ML.StandardTrainersCatalog.yml | 1 invalid cross reference(s) ""Microsoft.ML.Calibrators.PlattCalibratorTrainer"". |  
dotnet/api/Microsoft.ML.StandardTrainersCatalog.yml | 1 invalid cross reference(s) ""Microsoft.ML.Calibrators.PlattCalibratorTrainer"". |  
dotnet/api/Microsoft.ML.Trainers.FastTree.FastTreeBinaryTrainer.Options.yml | 1 invalid cross reference(s) ""Microsoft.ML.TreeExtensions.FastTree(Microsoft.ML.BinaryClassificationCatalog.BinaryClassificationTrainers,Microsoft.ML.Trainers.FastTree.FastTreeBinaryTrainer.Options)". |  
dotnet/api/Microsoft.ML.Trainers.FastTree.FastTreeBinaryTrainer.yml | 1 invalid cross reference(s) ""Microsoft.ML.TreeExtensions.FastTree(Microsoft.ML.BinaryClassificationCatalog.BinaryClassificationTrainers,Microsoft.ML.Trainers.FastTree.FastTreeBinaryTrainer.Options)". |  
dotnet/api/Microsoft.ML.Trainers.FastTree.FastTreeRankingModelParameters.yml | 1 invalid cross reference(s) ""Microsoft.ML.Trainers.FastTree.InternalTreeEnsemble"". |  
dotnet/api/Microsoft.ML.Trainers.FastTree.FastTreeRegressionModelParameters.yml | 1 invalid cross reference(s) ""Microsoft.ML.Trainers.FastTree.InternalTreeEnsemble"". |  
dotnet/api/Microsoft.ML.Trainers.FastTree.FastTreeTweedieModelParameters.yml | 1 invalid cross reference(s) ""Microsoft.ML.Trainers.FastTree.InternalTreeEnsemble"". |  
dotnet/api/Microsoft.ML.Trainers.FastTree.yml | 5 invalid cross reference(s) ""Microsoft.ML.TreeExtensions.FastTree(Microsoft.ML.BinaryClassificationCatalog.BinaryClassificationTrainers,Microsoft.ML.Trainers.FastTree.FastTreeBinaryTrainer.Options)", "Microsoft.ML.Trainers.FastTree.InternalQuantileRegressionTree", "Microsoft.ML.Trainers.FastTree.InternalRegressionTree", "Microsoft.ML.Trainers.FastTree.TreeEnsembleModelParametersBasedOnQuantileRegressionTree.CreateTreeEnsembleFromInternalDataStructure", "Microsoft.ML.Trainers.FastTree.TreeEnsembleModelParametersBasedOnRegressionTree.CreateTreeEnsembleFromInternalDataStructure". |  
dotnet/api/Microsoft.ML.Trainers.LightGbm.DartBooster.Options.yml | 1 invalid cross reference(s) ""Microsoft.ML.Trainers.LightGbm.Booster"". |  
dotnet/api/Microsoft.ML.Trainers.LightGbm.GossBooster.Options.yml | 1 invalid cross reference(s) ""Microsoft.ML.Trainers.LightGbm.Booster"". |  
dotnet/api/Microsoft.ML.Trainers.LightGbm.GradientBooster.Options.yml | 1 invalid cross reference(s) ""Microsoft.ML.Trainers.LightGbm.Booster"". |  
dotnet/api/Microsoft.ML.Trainers.LightGbm.LightGbmBinaryModelParameters.yml | 1 invalid cross reference(s) ""Microsoft.ML.Trainers.FastTree.InternalTreeEnsemble"". |  
dotnet/api/Microsoft.ML.Trainers.LightGbm.LightGbmRankingModelParameters.yml | 1 invalid cross reference(s) ""Microsoft.ML.Trainers.FastTree.InternalTreeEnsemble"". |  
dotnet/api/Microsoft.ML.Trainers.LightGbm.LightGbmRegressionModelParameters.yml | 1 invalid cross reference(s) ""Microsoft.ML.Trainers.FastTree.InternalTreeEnsemble"". |  
dotnet/api/Microsoft.ML.Trainers.LightGbm.yml | 1 invalid cross reference(s) ""Microsoft.ML.Trainers.LightGbm.Booster"". |  
dotnet/api/Microsoft.ML.Trainers.MatrixFactorizationTrainer.yml | 1 invalid cross reference(s) """"Microsoft.ML.Trainers.MatrixFactorizationTrainer.LossFunctionType.SquareLossOneClass"""". |  
dotnet/api/Microsoft.ML.Trainers.OnlineLinearOptions.yml | 1 invalid cross reference(s) ""Microsoft.ML.Trainers.OnlineLinearOptions.InitialWeights"". |  
dotnet/api/Microsoft.ML.Trainers.SdcaMaximumEntropyMulticlassTrainer.yml | 1 invalid cross reference(s) ""Microsoft.ML.Trainers.SdcaMulticlassTrainerBase"". |  
dotnet/api/Microsoft.ML.Trainers.SdcaNonCalibratedMulticlassTrainer.yml | 1 invalid cross reference(s) ""Microsoft.ML.Trainers.SdcaMulticlassTrainerBase"". |  
dotnet/api/Microsoft.ML.Trainers.SymbolicSgdLogisticRegressionBinaryTrainer.yml | 1 invalid cross reference(s) ""Microsoft.ML.Data.BinaryPredictionTransformer"". |  
dotnet/api/Microsoft.ML.Trainers.SymbolicSgdLogisticRegressionBinaryTrainer.yml | 1 invalid cross reference(s) ""Microsoft.ML.Data.BinaryPredictionTransformer"". |  
dotnet/api/Microsoft.ML.Transforms.KeyToValueMappingEstimator.yml | 1 invalid cross reference(s) ""Microsoft.ML.Data.AnnotationInfo"". |  
dotnet/api/Microsoft.ML.Transforms.KeyToVectorMappingEstimator.yml | 1 invalid cross reference(s) ""Microsoft.Ml.Data.KeyDataViewType"". |  
dotnet/api/Microsoft.ML.Transforms.TensorFlowModel.yml | 2 invalid cross reference(s) "Microsoft.ML.Transforms.TensorFlowModel.ModelPath", "Microsoft.ML.Transforms.TensorFlowModel.Session". |  
dotnet/api/Microsoft.ML.Transforms.TensorFlowModel.yml | 2 invalid cross reference(s) "Microsoft.ML.Transforms.TensorFlowModel.ModelPath", "Microsoft.ML.Transforms.TensorFlowModel.Session". |  
dotnet/api/Microsoft.ML.yml | 3 invalid cross reference(s) "Microsoft.ML.TrainContext", "Microsoft.ML.Data.CatalogUtils", "Microsoft.ML.IPredictor". |
